subsys/net/lib/nrf_cloud/src/nrf_cloud_fsm.c

```diff
@@ -480,65 +362,69 @@ static int handle_pin_complete(const struct nct_evt *nct_evt)
 
 static int cc_rx_data_handler(const struct nct_evt *nct_evt)
 {
+	__ASSERT_NO_MSG(nct_evt != NULL);
+	__ASSERT_NO_MSG(nct_evt->param.cc != NULL);
 	int err;
 	enum nfsm_state new_state;
 	const struct nrf_cloud_data *payload = &nct_evt->param.cc->data;
-	bool config_found = false;
-	bool control_found = false;
 	const enum nfsm_state current_state = nfsm_get_current_state();
-
-	LOG_DBG("CC RX on topic %s: %s",
+	struct nct_cc_data msg = {
+		.opcode = NCT_CC_OPCODE_UPDATE_REQ,
+		.message_id = NCT_MSG_ID_STATE_REPORT
+	};
+	struct nrf_cloud_topic *t = &nct_evt->param.cc->topic;
+	int dlen = strlen(NRF_CLOUD_JSON_KEY_DELTA);
+	bool is_delta = (strncmp(&((const char *)t->ptr)[t->len - dlen],
```

Comment:
And why is there an nct_cc_rx_opcode_map and not also an ncr_cc_tx_opcode_map, since this map is used for both RX and TX lists?

I don't see that any part of sdk-nrf uses the RX opcode, but the TX opcode is used.

I'll add a tx map and fix this up.

Reply:
Nevermind.  I'll create ticket to follow up on that.

Reply:
yeah i think the rx/tx topic op codes and lists could be re-worked, so probably better to do in a separate ticket.
it is very hard to follow.

Reply:
https://nordicsemi.atlassian.net/browse/IRIS-7258